pkg/common/backward_compability.go

```diff
@@ -0,0 +1,30 @@
+package common
```

Comment:
This file was added for add any backward compatibility code.
In this case with new thor3 we have moved the API response from task progress to report floating point number, previously it was integer from 1-100 now it will 0.0 to 1 in range.